cli/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+All changes to C2PA Tool are documented in this file.  For additional information on versions 0.9.x and earlier, see the [Archived release motes](../docs/release-notes.md).
```

Comment:
Why segregate like this?

Reply:
It was simply because the changelog was added in addition to the existing manually-written release notes.  Do you want me to add the content of that file to the end of this one?  I suspect very few people will care about that info... so I don't have a strong opinion.